about.css

```diff
@@ -0,0 +1,466 @@
+  margin: 25% auto;
+}
+
+.home-content {
+  background: #fff;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 25px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: -0.06em;
+  color: #000;
+  padding: 24px 10px;
+  width: 100%;
+}
+
+.home-date {
+  font-style: normal;
+  font-weight: 800;
+  font-size: 12px;
+  line-height: 25px;
+  text-align: center;
+  letter-spacing: 0.02em;
+  text-decoration-line: underline;
+  color: #5f5f5f;
+}
+
+.at {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 25px;
+  text-align: center;
+  color: #7a7a7a;
+  margin: 0 44px;
+}
+
+.info {
+  text-align: center;
+}
+
+.info h2 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 28px;
+  letter-spacing: -0.02em;
+  color: #2c2c2c;
+  margin: 20px 74px 5px;
+}
+
+.info p {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 21px;
+  letter-spacing: -0.03em;
+  color: #838383;
+  margin: 20px 10px;
+}
+
+.about_logo {
+  margin-bottom: 20px;
+}
+
+.abt-img > img:nth-child(2) {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.partner {
+  text-align: center;
+  background-color: #414246;
+  color: #fff;
+}
+
+.partner h2 {
+  margin: 0;
+}
+
+.img-logo {
+  padding-bottom: 25px;
+}
+
+.img-logo ul {
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0;
+  justify-content: space-around;
+  align-items: center;
+  margin: 0 20px;
+}
+
+.img-logo li {
+  list-style: none;
+  padding: 10px;
+}
+
+.footer {
+  display: flex;
+  margin: 0 18px;
+  gap: 15%;
+  padding: 20px;
+}
+
+.footer-des {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 10px;
+  line-height: 13px;
+  letter-spacing: -0.01em;
+  color: #3b3e45;
+  width: 50%;
+  margin: 0;
+}
+
+.hidden {
+  display: none;
+}
+
+.menu-popup {
+  background-color: gray;
+  background-size: cover;
+  background-repeat: no-repeat;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  mix-blend-mode: multiply;
+  display: flex;
+  flex-direction: column;
+  backdrop-filter: blur(6px);
+  align-items: start;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.banner {
+  display: none;
+}
+
+.toolbar2 {
+  display: none;
+}
+
+.home-content .desktop {
+  display: none;
+}
+
+#desktop {
+  display: none;
+}
+
+#underline {
+  display: none;
+}
+
+section {
+  border: 1px solid #fafafa;
+  box-shadow: 0 2px 0 rgba(0, 0, 0, 0.25);
+}
+
+#visibile {
+  display: flex;
+}
+
+#hidden {
+  display: none;
+}
+
+div.abt-img {
+  width: 100%;
+}
+
+@media only screen and (max-width: 768px) {
+  .showcontent {
+    background-color: #3b3e45;
+    background-size: cover;
+    color: #fff;
+    background-repeat: no-repeat;
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    mix-blend-mode: multiply;
+    display: flex;
```

Comment:
[ Optional ] I think it would be better if the mobile menu is hidden by default using the CSS `display: none` property. Currently, when I switch to the mobile version the mobile menu is displayed by default even before  I click on the hamburger menu. Using `display: none` will hide the menu by default such that the only way to open the mobile menu will be by clicking on the hamburger menu.